src/handlers/messageHandler.ts

```diff
@@ -94,13 +109,10 @@ async function autoCreateThread(message: Message, requestId: Snowflake) {
 	});
 
 	const creationDate = message.createdAt.toISOString().slice(0, 10);
-	const authorName = authorMember === null || authorMember.nickname === null
-		? authorUser.username
-		: authorMember.nickname;
+	const authorName =
+		authorMember === null || authorMember.nickname === null ? authorUser.username : authorMember.nickname;
 
-	const name = emojisEnabled(guild)
-		? `🆕 ${authorName} (${creationDate})`
-		: `${authorName} (${creationDate})`;
+	const name = emojisEnabled(guild) ? `🆕 ${authorName} (${creationDate})` : `${authorName} (${creationDate})`;
```

Comment:
I've been looking for a very long time to find the rule that changed this. I don't want any ternaries that are one-lined, they should always look like this:
```ts
const name = emojisEnabled(guild)
	? `🆕 ${authorName} (${creationDate})`
	: `${authorName} (${creationDate})`;
```

This seems like it should be enforced already by prettier, just browsing examples such as [this one](https://prettier.io/docs/en/rationale.html#jsx). I don't know if it's overridden by some ESLint thing? I can't figure it out for the life of me, spent over an hour just trying to figure out why it's not indenting properly. Do you have any idea @nchristopher?

Reply:
For reference, it seems like prettier's actual behavior is to always newline the ternaries. See [this playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEA5gE5xwwAU6AznMQJT7AA6U++pM6xbNdr7dgH58AAwDqcADaQAtnHwAjAIZgA1gBp8AEmB9iAOijL5AXwCEogYKT5mIAOKlyOKIWpaYxZQDdpdc3wAZRxCXgAHfBgIABNlAE9zewBuVlNWEA0QCHDcaGpkUGViYggAdwAFYoQClGUpMoSCrMVvdXIgkzgAGVc4ZFR62ha2tQ7w1VdCZC90OCy4WUU4GJiV7uU3dGVCOAAxCGJZZRhcN2QQZXRozJAACxhZKXE7nHhqCbA4IJq3nB83vELmBqM0QK5aMQYBVvIRjgMhvMQAAragADxCbikcAAiugIPAEVJhiAJsRIRcVMspLdwsRXDBxDgYjA7sgABwABiydIgtHE3nCFzpcEhfluAEd8fAYTlapdqABaKBkNYxW6kKU4UgwnbwpCDYlI2iyHAzYhzLLUKbYvEE-oGxFZGDKRRMllspAAJmd3hwUimAGEILJ9SBRQBWW58AAqrtqhpJPjmAEkoGtYEEwPTcgBBdNBGDxbFE2imUxAA). What have we configured to mess this up?

Reply:
Ah, I think I figured it out. It will always newline when the ternary would otherwise go over the print width, see [this updated playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzArlMMCW0AEA5gE5xwwAU6AznMQJT7AA6U++kU1M+UAhgFs4+ALz44AiACsc1AKL8ARgBs4AEwqF0OZWvqtmMQwH58AA0A8G4FTd-ABJgfdDAAWEYgDlBcAL74K9sFI+XGgAEWCfejMDIxgkc3tHFzdPIV9-YEC4YLwocPhvKIBuVnZSGHRiNho6UvZ8UzMAdThlSCF8RT4wAGsAGjtgGuIAOn40gEJotnr45hAAcVJyHChCagGYYj4AN1a6CfwAZRxCaoAHfBgINT4ATwn5kqhvVhA+kAhzkK5kUD5iMQIAB3AAKAIQ1GQID4ymB9yhH0U2165COXgAMqs4MhULDaEiUT00eduqtCMgtug4B8JIp1Gp1Bi+Gt0HxCHAAGJuATBXBraFJCDvEDOGACZRNZw4eDUUlgOBHSEynA7GV3aFgaiIkCrWjEGCg7aEXm4-E0kBSagADxOa1UAEV0BB4GblASQKTiProV16coRediKsYE0cGoXMgAIxRgAMHyDEFoTW252hQbg+r2IoAjs74EavlCUHxqABaKBkRlqEWkPM4UhG9mmpB490W2gCHCU4jUj7UcmO-M41vmj4wPiKMMR5zIABM4+2OnJAGEIAIWyBMwBWEU1AAqk+LbY9O2pAEkoIzYEdAjhvgBBK9HGB3VRu2jebxAA). I will just go ahead and merge then 😄